internal/cluster/cluster.go

```diff
@@ -999,6 +999,10 @@ func (m Manager) PermanentClustersDeletion(ctx context.Context, olderThan strfmt
 			deleteFromDB = false
 			m.log.WithError(err).Warnf("Failed deleting s3 manifests of cluster %s", c.ID.String())
 		}
+		if _, err := objectHandler.DeleteObject(ctx, c.ID.String()); err != nil {
```

Comment:
Add UT

Reply:
The existing unit-test already covered this, but I changed the expectation to be more explicit for this call.